### PR TITLE
Adding more specific instruction regarding NPM version

### PR DESCRIPTION
**Pull request by dannywillems**
If using v19, we get errors. Nice to have instructions with the right version to use + how to install.

### CODE REVIEW

File: docs/node-developers/style-guide.mdx [hunk 1]
```diff
@@ -185,7 +185,7 @@ val gen :
 
 ### Typesafe invariants (help with naming this section)
 
-Often times in Mina, we need to perform very important checks on certain pieces of data.
+Oftentimes in Mina, we need to perform very important checks on certain pieces of data.
```
Comment (barriebyron) [inline docs/node-developers/style-guide.mdx:188]:
```suggestion
In Mina, very important checks are frequently performed on certain pieces of data.
```

**Review by barriebyron — APPROVED** [review]
thank you @dannywillems  for this update. See my suggested change for wording (I love words!)

File: README.md [hunk 2]
```diff
@@ -14,7 +14,16 @@ The docs output is generated by a CI pipeline which means that we can update the
 
 During the documentation build process, each PR goes through Vercel CI to test the doc updates and build a preview of your changes. Vercel for GitHub automatically deploys and provides preview deployment URLs. After the PR status is ready, click **Visit Preview** and navigate to the changed page to see your doc updates.
 
-Sometimes it is helpful to see local updates before you push changes to GitHub. To run a local build:
+Sometimes it is helpful to see local updates before you push changes to GitHub.
+<!-- In case of version change, update .github/workflows/{test-tutorials.yml, snarkjs-api-reference.md} accordingly -->
+Local builds require NPM, version 16 precisely.
```
Comment (ymekuria) [inline README.md:19]: I think you mean node instead of NPM here right?

Comment (dannywillems) [reply README.md]: Updated, thanks.

**Review by ymekuria — APPROVED** [review]

**Pull request merged**